lib/paneron/register/data_set_metadata/stakeholder.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require "lutaml/model"
+
+module Paneron
+  module Register
+    class DataSetMetadata < Lutaml::Model::Serializable
+      class Stakeholder < Lutaml::Model::Serializable
+        attribute :name, Lutaml::Model::Type::String
+        attribute :roles, Lutaml::Model::Type::String,
+                  collection: true,
+                  values: %w[
+                    owner
+                    control-body
+                    control-body-reviewer
+                    manager
+                    submitter
+                  ]
+        attribute :gitServerUsername, Lutaml::Model::Type::String
+        attribute :affiliations, Paneron::Register::DataSetMetadata::Affiliations
+        # NOTE: There is no { optional: true } # Use render_nil: true for all other attributes in the render block instead
```

Comment:
@ribose-jeffreylau I don't quite understand this comment. If there is a feature request / issue can you please file at lutaml-model? Thanks!

Reply:
I was thinking from the perspective of Typescript, since RegistryKit is where I model this after.
I can get rid of that comment.

Reply:
Actually, for the sake of keeping close to RegistryKit, I might add cardinality requirements instead.

Reply:
Cardinality is only defined for collections.  I'll leave the lutaml attributes alone for now...

Reply:
We didn't call it "cardinality" but here's the doc:
* https://github.com/lutaml/lutaml-model#attribute-as-a-collection

```ruby
attribute :name_of_attribute, Type, collection: true
attribute :name_of_attribute, Type, collection: {min}..{max}
attribute :name_of_attribute, Type, collection: {min}..
```

Reply:
Thanks!  I have read that part.  The idea of declaring certain fields as "required" by declaring it a collection makes it feel... I'll leave that for a separate issue :)

Reply:
#16